src/encoded/audit/file.py

```diff
@@ -5,6 +5,27 @@
 
 from .item import STATUS_LEVEL
 
+def check_award_condition(value, awards):
+    return value.get('award', None) and value.get('award')['rfa'] in awards
```

Comment:
Perhaps `value.get('award', {}).get('rfa')` in case the award doesn't have an rfa for whatever reason.

Reply:
fixed